tutorial/Lab2/README.md

```diff
@@ -0,0 +1,151 @@
+   redis-slave-deployment.yaml
+   ```
+
+2. Delete the Redis slave service and pods:
+
+   ```console
+   $ kubectl delete svc redis-slave --namespace default
+   service "redis-slave" deleted
+   $ kubectl delete deployment redis-slave --namespace default
+   deployment.extensions "redis-slave" deleted
+   ```
+
+3. Update the guestbook service from `LoadBalancer` to `NodePort` type:
+
+   ```$ sed -i.bak 's/LoadBalancer/NodePort/g' guestbook-service.yaml```
+
+   Note: Like in Step 1., you may want to archive before making the changes..
+   
+4. Delete the guestbook service:
+    
+    ```$ kubectl delete svc guestbook --namespace default```
+    
+5. Recreate the service with `NodePort` type:
+
+    ```$ kubectl create -f guestbook-service.yaml```
+    
+6. To check the updates, you can run ```kubectl get all --namespace default```:
+    
+    ```console
+    NAME                                READY     STATUS    RESTARTS   AGE
+    pod/guestbook-v1-7fc76dc46-9r4s7    1/1       Running   0          1h
+    pod/guestbook-v1-7fc76dc46-hspnk    1/1       Running   0          1h
+    pod/guestbook-v1-7fc76dc46-sxzkt    1/1       Running   0          1h
+    pod/redis-master-5d8b66464f-pvbl9   1/1       Running   0          1h
+    
+    NAME                   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    service/guestbook      NodePort    172.21.45.29    <none>        3000:31989/TCP   31s
+    service/kubernetes     ClusterIP   172.21.0.1      <none>        443/TCP          9d
+    service/redis-master   ClusterIP   172.21.232.61   <none>        6379/TCP         1h
+    
+    NAME                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/guestbook-v1   3         3         3            3           1h
+    deployment.apps/redis-master   1         1         1            1           1h
+    
+    NAME                                      DESIRED   CURRENT   READY     AGE
+    replicaset.apps/guestbook-v1-7fc76dc46    3         3         3         1h
+    replicaset.apps/redis-master-5d8b66464f   1         1         1         1h
+    ```
+    Note: The service type has changed (to `NodePort`) and a new port has been allocated (`31989` in this output case) to the guestbook 
+    service. All `redis-slave` resources have been removed.
+
+5. View the guestbook as per [Lab1](../Lab1/README.md), using the updated port for the guestbook service
+   
+# Update the application using Helm
+
+In this part of the lab we will update the previously deployed application `guestbook-demo`, using Helm.
+
+Before we start, let's take a few minutes to see how does Helm simplify the process compared to using Kubernetes directly. Helms' use of a [template language](https://docs.helm.sh/chart_template_guide/) provides great flexibility and power to chart authors which removes the complexity to the chart user. In the guestbook example, we are using the following capabilities of templating:
```

Comment:
Change to ```Before we start, let's take a few minutes to see how Helm simplifies the process compared to using Kubernetes. ```(do you need to say directly? or can it be assumed without?) ```Helm's use of a [template language](https://docs.helm.sh/chart_template_guide/) provides great flexibility and power to chart authors, which removes the complexity to the chart user. In the guestbook example, we'll use the following capabilities of templating:```

Reply:
updated